### PR TITLE
Char count and Text Color

### PR DESCRIPTION
**Pull request by shah743**
Modified ReadMe

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -28,18 +28,20 @@ Add this to your application.js after page is ready:
...
 
-And finally, in your views, tag your *<img/>* class with `initialjs-avatar` class. You can use *data-attributes* as described in [initial.js](https://github.com/mozilla/metrics-graphics). Or you can use the helper below.
+And finally, in your views, tag your *<img/>* class with `initialjs-avatar` class. You can use *data-attributes* as described in [initial.js](https://github.com/judesfernando/initial.js). Or you can use the helper below.
 
 Enjoy.
 
 ## ActionView Helper
 
-For extra simplicity, you can use this helper in your view to generate your avatar image from your model. In the following examples we'll use User, but any model responding to `:name` will do. You can learn the options available and their defaults directly from the examples:
+For extra simplicity, you can use this helper in your view to generate your avatar image from your model. In the following examples we'll use User, but any model responding to `:full_name` will do. You can learn the options available and their defaults directly from the examples:
```
Comment (dgilperez) [inline README.md]: This is a breaking change. Please split into two commits.

File: lib/initialjs-rails/view_helpers.rb [hunk 2]
```diff
@@ -7,16 +7,32 @@ def avatar_image(avatarable, options = {})
...
+        'text-color': txt_color
+      }
 
       data_attributes.merge!(radius: (size * 0.13).round) if round_corners
 
-      content_tag :img, nil, alt: avatarable.name, class: "initialjs-avatar #{klass}".strip, data: data_attributes
+      content_tag :img, nil, alt: avatarable, class: "initialjs-avatar #{klass}".strip, data: data_attributes
+    end
+
+    protected
```
Comment (dgilperez) [inline lib/initialjs-rails/view_helpers.rb]: Why protected?

File: README.md (hunk 1 above)
Comment (dgilperez) [inline README.md]: I meant two PRs.

File: lib/initialjs-rails/view_helpers.rb [hunk 3]
```diff
@@ -7,16 +7,32 @@ def avatar_image(avatarable, options = {})
...
+
+    def get_name(name, count)
```
Comment (dgilperez) [inline lib/initialjs-rails/view_helpers.rb]: What if count is 3, or larger?

File: README.md (hunk 1 above)
Comment (dgilperez) [inline README.md]: In the second PR, I'd like to support old behavior as well. Something like 

name = avatarable.responds_to?(:name) ? avatarable.name : avatarable

**Comment by dgilperez** [comment]
Thanks for the PR!

File: README.md (hunk 1 above)
Comment (shah743) [inline README.md]: hmmm ok i'll updated code with old behavior

File: lib/initialjs-rails/view_helpers.rb (hunk 3 above)
Comment (shah743) [inline lib/initialjs-rails/view_helpers.rb]: there is if condition if count will be 3 or higher they will show char from full name

**Comment by dgilperez** [comment]
Thanks! 

I have bumped the version, but I'm waiting to test this before releasing it to rubygems. No time until tomorrow, I think.

Can you check if the current master works for you as expected?

**Pull request merged**